mpconstants/countries.py

```diff
@@ -79,7 +91,7 @@
     },
     'CHN': {
         'adolescent': True,
-        'default_currency': 'RMB',
+        'default_currency': 'CNY',
```

Comment:
`'RMB'` is not a valid currency according to https://en.wikipedia.org/wiki/ISO_4217, and it did not match an existing currency in `payments.py`.